cardano-cli/src/Cardano/CLI/Byron/Delegation.hs

```diff
@@ -12,34 +12,34 @@ module Cardano.CLI.Byron.Delegation
   )
 where
 
-import           Cardano.Api.Byron
+import Cardano.Api.Byron
 
+import Cardano.CLI.Byron.Key (ByronKeyFailure, renderByronKeyFailure)
+import Cardano.CLI.Types.Common (CertificateFile (..))
 import qualified Cardano.Chain.Delegation as Dlg
-import           Cardano.Chain.Slotting (EpochNumber)
-import           Cardano.CLI.Byron.Key (ByronKeyFailure, renderByronKeyFailure)
-import           Cardano.CLI.Types.Common (CertificateFile (..))
-import           Cardano.Crypto (ProtocolMagicId)
+import Cardano.Chain.Slotting (EpochNumber)
```

Comment:
This feels like a decrease of readability for me. Maybe we could enable [`ImportQualifiedPost` globally](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/import_qualified_post.html) to mitigate that?